services/web/project/__init__.py

```
@@ -0,0 +1,34 @@
+from flask import Flask
+import os
+from project.config_sw import swaggerui_blueprint
+from flask_cors import CORS
+from project.models import db
+from project.routes import main
+from project.swagger import swagger
+
+
+def create_app(database_url=os.getenv("DATABASE_URL", "sqlite://")):
```

Ruff (0.8.2)

Do not perform function call `os.getenv` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)